src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java

```diff
@@ -103,5 +101,27 @@ public AccountManagementSteps() {
                         assertThat(secondAccount.getBalance())
                                 .isEqualByComparingTo(new BigDecimal(newBalance)));
 
+        Given("^a customer has an account with balance (\\d+)$", (Integer balance) -> {
+            firstAccount = new Account(customer);
+            firstAccount.setBalance(new BigDecimal(20));
+            firstAccount.setBalance(new BigDecimal(balance));
+            bankProductsRepository.addAccount(firstAccount);
+        });
+
+        When("^he opens a deposit with balance (\\d+)$", (Integer depositBalance) -> {
+            firstDeposit = bank.openDeposit(customer, firstAccount, new BigDecimal(depositBalance));
+            bankProductsRepository.addDeposit(firstDeposit);
+        });
+ ;
+        Then("^he owns a deposit with balance (\\d+)$", (Integer depositBalance) -> {
+            customerDeposits = bank.getDepositsForCustomer(customer);
+            assertThat(customerDeposits).contains(firstDeposit);
+            assertThat(firstDeposit.getBalance()).isEqualByComparingTo(new BigDecimal(depositBalance));
+        });
+
+        And("^the account has balance (\\d+)$", (Integer accountNewBalance) -> {
+            //assertThat(firstAccount.getBalance()).isEqualByComparingTo(new BigDecimal(accountNewBalance));
```

Comment:
Uncomment this assertion. If it's failing try to investigate why.

Reply:
Done. I have no clue why I left it commented.